src/common/components/forms/DownloadableNFTCard.tsx

```diff
@@ -5,25 +5,21 @@ import FeatherIcon from 'feather-icons-react';
 import * as htmlToImage from 'html-to-image';
 import { shortenAddress } from '../../../modules/utils/string';
 import { DisplaySOL } from '../CurrencyHelpers';
-import { Listing, Marketplace, Nft, Offer } from '@holaplex/marketplace-js-sdk';
+import { Listing, Nft } from '@holaplex/marketplace-js-sdk';
```

Comment:
Good catch, i believe I had those in for once we actually need to offer stuff but its related to #227 which is a longer ways out but easy enough to add back in